SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialLoadBalanceEnforcer.cpp

```diff
@@ -136,3 +192,11 @@ TArray<SpatialLoadBalanceEnforcer::AclWriteAuthorityRequest> SpatialLoadBalanceE
 
 	return PendingRequests;
 }
+
+bool SpatialLoadBalanceEnforcer::CanEnforce(Worker_EntityId EntityId)
```

Comment:
This is effectively the "is this entity complete" predicate for the load balance enforcer